examples/azure-rules-perf-optimized.yaml

```diff
@@ -0,0 +1,21 @@
+---
+controls:
+version: 1.0.0
+id: 2
+description: "HA Checker for SAPHanaSR Scale-up Perf-Optimized"
+type: "master"
+groups:
+  - id: 2.1
+    description: "Check OS and package versions are supported and up-to-date"
+    checks:
+      - id: 2.1.1
+        description: "Check the base product"
+        # Assume the perf-optimized scenario and
+        # check only that the base probuct is always sles_sap.prod (case-insensitively)
+        audit: 'readlink /etc/products.d/baseproduct | sed -e "s/\(.*\)/\L\1/"'
```

Comment:
I think the `| sed ...` is  unnecessary here.

Reply:
I was told that the base product could be in both lower- and upper-case.

Reply:
we're reading the target of the symlink here, and we assume case sensitive filesystems. the check by itself is not very precise anyway, as we're just checking for filename not filecontents (and in the products.d specification, filename doesn't matter, file content is all that matters). 

then we need to get back to checking the action product version and name again, like suggested earlier.